hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -69,4 +63,4 @@ The Raspberry Pi 4's bootcode is stored in [EEPROM](../booteeprom.md) and can be
 
 - The default timeout for checking bootable USB devices is 2 seconds. Some flash drives and rotational harddrives power up too slowly. It’s possible to extend this timeout to five seconds (add a new file `timeout` to the SD card), but there are devices that fail to respond within this period as well.
 - Some flash drives have a very specific protocol requirement that is not handled by the bootcode and may thus be incompatible.
-- 3.5" HDD's commonly require 12V as well as 5V and may draw too much current for the Pi's USB connection. Use an externally powered 3.5" HDD in this case.
+- Lack of power can be an issue, as some devices draw too much current from the Raspberry Pi's USB connection and fail to start, or prevent the Raspberry Pi from booting. An external power supply for the device will be required in these cases.
```

Comment:
I think you need to mention a powered USB hub here as well.

Reply:
"An external power supply or powered USB hub will be..." ?

Reply:
> "An external power supply or powered USB hub will be..." ?

That would be OK. Usually devices either have their own PSU or use bus power, so I would go with something like:

"Lack of power can be an issue, so it is recommended to use a powered USB hub, particuarly if you are attaching more than one storage device to the Raspberry Pi. If your device has its own power supply then be sure to use it.